rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [x] when you set up many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental, **A rental belongs to a book and also belongs to a person:** so Kindly I recommend you to set  `Getters and Setters for @book and @person instance variables` just like the below sample instead of using `attr_reader `, Kindly Fix it.

```
attr_accessor :date, :book, :person

```